packages/integration/lib/src/widgets/default/default_widgets.dart

```diff
@@ -288,7 +288,7 @@ class LMFeedDefaultWidgets {
       postId: post.id,
       style: feedThemeData.mediaStyle,
       carouselIndicatorBuilder:
```

Comment:
What is the use of this file now?

Reply:
it is used to create default widget for every screen, if we remove this class then we have to create default post for each and every screen, since a lots of widgets are reused in multiple screen, i.e: post, comment. we should keep it

Reply:
Got it